packages/ui/components/form/select/components.tsx

```diff
@@ -23,6 +23,7 @@ export const InputComponent = <
         inputClassName
       )}
       {...props}
+      aria-label="Timezone Selection"
```

Comment:
his is not limited only to TimezoneSelect, so this label name is not suitable. And Instead, update and try to change the `TimezoneSelect` component

Reply:
Thank you both for your feedback, @mfranzke, @Praashh  and @anikdhabal . I understand that adding the aria-label directly to a general component would affect all select elements across the application. I'll make the necessary adjustments to ensure the label is appropriately scoped to the TimezoneSelect component. I'll update the PR accordingly.

Thanks again for the guidance!